oracle/orchestrator/market_mapper.go

```diff
@@ -55,6 +55,7 @@ func (o *ProviderOrchestrator) listenForMarketMapUpdates(ctx context.Context) {
 			}
 
 			// Write the market map to the configured path.
+			o.logger.Info("writing updated market map", zap.String("file", o.writeTo))
```

Comment:
can we move this into the `WriteMarketMap` function since the o`writeTo` might be empty?